src/main/java/gov/usgs/wma/waterdata/collections/CollectionsController.java

```diff
@@ -129,19 +136,59 @@ public String getOgcCollectionFeatures(
 							content=@Content())
 			},
 			externalDocs=@ExternalDocumentation(url="http://docs.opengeospatial.org/is/17-069r3/17-069r3.html#_feature_")
-			)
+		)
 	@GetMapping(value = "collections/{collectionId}/items/{featureId}", produces = MediaType.APPLICATION_JSON_VALUE)
 	public String getOgcCollectionFeature(
 			@RequestParam(value = "f", required = false, defaultValue = "json") String mimeType,
 			@PathVariable(value = "collectionId") String collectionId,
 			@PathVariable(value = "featureId") String featureId, HttpServletResponse response) {
 
-		String rtn = collectionsDao.getCollectionFeatureJson(collectionsParams.buildParams(collectionId, featureId));
-		if (rtn == null) {
-			response.setStatus(HttpStatus.NOT_FOUND.value());
-		}
+		return resultOr404(response, 
+				collectionsDao.getCollectionFeatureJson(collectionsParams.buildParams(collectionId, featureId)));
+	}
 
-		return rtn;
+	@Operation(
+			description="Return GeoJSON Data specific to the requested Monitoring Location and Time Series.",
+			responses= {
+					@ApiResponse(
+							responseCode="200",
+							description="GeoJSON representation of the Time Series.",
+							content=@Content(schema=@Schema(implementation=TimeSeriesGeoJSON.class))),
+					@ApiResponse(
+							responseCode="404",
+							description="The requested Time Series was not found.",
+							content=@Content())
+			},
+			externalDocs=@ExternalDocumentation(url="https://github.com/opengeospatial/omsf-profile/tree/master/omsf-json")
+		)
+	@GetMapping(value="collections/{collectionId}/items/{featureId}/time-series/{timeSeriesId}", produces=MediaType.APPLICATION_JSON_VALUE)
+	public String getTimeSeries(
+			@PathVariable(value="collectionId") String collectionId, // ex: networkId,
+			@PathVariable(value="featureId") String featureId, // ex: monitoringLocationId
+			@PathVariable(value="timeSeriesId") String timeSeriesId, //ex: USGS-123456
+			HttpServletResponse response) {
+		
+		// verify the collection and feature exist before fetching the time series
+		return resultOr404(response, timeSeriesDao.getTimeSeries(collectionId, featureId, timeSeriesId));
 	}
 
+	
+	/**
+	 * Helper method to set the response to 404 if there is no result from the request.
```

Comment:
Please update documentation

Reply:
What would you like updated, that it will not set it if it has been set?

Reply:
@param lambda

Reply:
ok